### ISSUE TITLE
Provide search + filter options on https://install.appcenter.ms/apps

### ISSUE THREAD

**Issue by Brantone** [microsoft/appcenter#2217]
Recreating to keep https://github.com/microsoft/appcenter/issues/1626 relevant.

**Describe the solution you'd like**
Simplistic filter for OS or Release Type on https://install.appcenter.ms/apps
Essentially same things as https://appcenter.ms/apps

**Describe alternatives you've considered**
Using browser search; however, that's not a great UX when viewing on mobile device (where only "Release Type" would be needed).

**Additional context**
We have almost hundred binaries, so finding the necessary build gets cumbersome. As an admin/developer, I can filter the builds; however, for distribution groups this is not possible. Would be nice to make it easier on distro users, if they could do simple filters.

Thanks very much for your consideration and hard work.

**Comment by Brantone** [comment]
<bump>

**Comment by Brantone** [comment]
So extremely very useful.

**Comment by stianjensen** [comment]
Agreed! Just getting the option to filter by distribution group would be very helpful. Without that we'll probably need to look for alternative distribution methods, because finding what we need is simply too cumbersome.

(writing this as a new comment since you keep marking useful issues as stale otherwise)

**Comment by Brantone** [comment]
Hey Microsoft ... how about even opening up some parts to allow for open source contribution like on other repos.

**Comment by Brantone** [comment]
Still relevant

**Comment by stianjensen** [comment]
Here's some activity.

**Comment by Brantone** [comment]
Bump

**Comment by pf-mcapps** [comment]
Yes, this basic functionality is needed for companies that have produced more than a few apps.  

Another way to help alleviate the problem of having to sort through too many apps would be to allow an app to be marked as archived or hidden so it doesn't show up by default.  This way all of the prototypes and retired apps that a company creates would not have to be sifted past, but could still be quickly brought back when needed.

**Comment by giuseppeturelli** [comment]
Bump, that's a must have when you have many projects in parallel.

**Comment by Brantone** [comment]
😒

**Comment by stianjensen** [comment]
Congratulations on the new auto-closing bot!

Here's some more activity.

**Comment by matt-smith-kr** [comment]
How about this gets some love?
